src/components/NewMessageModal.vue

```diff
@@ -94,7 +94,10 @@ export default {
 				this.$store.commit('removeMessage', { id: data.draftId })
 
 				// Fetch new draft envelope
-				await this.$store.dispatch('fetchEnvelope', id)
+				await this.$store.dispatch('fetchEnvelope', {
```

Comment:
`fetchEnvelope` takes an id. you can't pass an object like this

Reply:
Why not? The action was refactored to `async fetchEnvelope({ commit, getters }, { accountId, id })`.